src/data/polynomial/erase_lead.lean

```diff
@@ -160,10 +160,11 @@ end erase_lead
 required to be at least as big as the `nat_degree` of the polynomial.  This is useful to prove
 results where you want to change each term in a polynomial to something else depending on the
 `nat_degree` of the polynomial itself and not on the specific `nat_degree` of each term. -/
-lemma induction_with_nat_degree_le {R : Type*} [semiring R] {P : polynomial R → Prop} (N : ℕ)
+lemma induction_with_nat_degree_le {R : Type*} [semiring R] (P : polynomial R → Prop) (N : ℕ)
```

Comment:
Could you please update the docstring?

Reply:
Since the docstring did not get into too many details about the statement, the current version still applies.  If you think that it would be better, I can make the docstring longer and possibly get to the improvement.

Do you think that it would be worth it?  It might take longer to read the docstring than to parse the statement of the lemma.